src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -59,33 +59,8 @@
 import net.sourceforge.kolmafia.persistence.SkillDatabase;
 import net.sourceforge.kolmafia.preferences.Preferences;
 import net.sourceforge.kolmafia.request.FamTeamRequest.PokeBoost;
-import net.sourceforge.kolmafia.session.BanishManager;
+import net.sourceforge.kolmafia.session.*;
```

Comment:
I believe convention is not to do this

Reply:
My IDE did this without my consent or knowledge; rolling it back